kong/plugins/ai-proxy/handler.lua

```diff
@@ -432,8 +425,9 @@ function _M:access(conf)
   -- now re-configure the request for this operation type
   local ok, err = ai_driver.configure_request(conf_m)
   if not ok then
-    kong.ctx.shared.skip_response_transformer = true
-    return internal_server_error(err)
+    kong_ctx_shared.skip_response_transformer = true
+    kong.log.err("failed to configure request for AI service: ", err)
+    return kong.response.exit(500)
```

Comment:
Should we response with the error message?

```suggestion
    return kong.response.exit(500, { error = { message = err } })
```

Reply:
No, this is an internal server error. Targeted at the admin of the system (hence the log line), it is of no concern for the end user.